src/Models/Question.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Models;
+
+use Engelsystem\Models\User\User;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property integer $id
+ * @property integer $enquirer_id
+ * @property string $question
+ * @property string $answer
+ * @property integer $answerer_id
+ * @property-read \Engelsystem\Models\User\User $answerer
+ * @property-read \Engelsystem\Models\User\User $enquirer
+ * @method static Builder|\Engelsystem\Models\Question newModelQuery()
+ * @method static Builder|\Engelsystem\Models\Question newQuery()
+ * @method static Builder|\Engelsystem\Models\Question query()
+ * @method static Builder|\Engelsystem\Models\Question whereAnswer($value)
+ * @method static Builder|\Engelsystem\Models\Question whereAnswererId($value)
+ * @method static Builder|\Engelsystem\Models\Question whereEnquirerId($value)
+ * @method static Builder|\Engelsystem\Models\Question whereId($value)
+ * @method static Builder|\Engelsystem\Models\Question whereQuestion($value)
+ */
+class Question extends BaseModel
+{
+    /** @var string[] */
+    protected $fillable = [
+        'enquirer_id',
+        'question',
+        'answerer_id',
+        'answer',
+    ];
+
+    /** @var string[] */
+    protected $casts = [
+        'enquirer_id' => 'integer',
+        'answerer_id' => 'integer',
+    ];
+
+    /**
+     * @return BelongsTo
+     */
+    public function enquirer(): BelongsTo
+    {
+        return $this->belongsTo(User::class, 'enquirer_id');
+    }
+
+    /**
+     * @return BelongsTo
+     */
+    public function answerer(): BelongsTo
+    {
+        return $this->belongsTo(User::class, 'answerer_id');
+    }
+
+    /**
+     * @return Builder
+     */
+    public static function findUnAnswered(): Builder
```

Comment:
I would name it `findNotAnswered` but thats only my personal preference

Reply:
Event shorter `unanswered` ;)